vendor/bat-native-ledger/src/bat/ledger/internal/media/twitter.cc

```diff
@@ -216,22 +216,28 @@ std::string MediaTwitter::GetShareURL(
   auto comment = args.find("comment");
   auto name = args.find("name");
   auto tweet_id = args.find("tweet_id");
+  auto hashtag = args.find("hashtag");
   if (comment == args.end() || name == args.end() || tweet_id == args.end())
     return std::string();
 
+  // Append hashtag to comment ("%20%23" = percent-escaped space and
+  // number sign)
+  std::string comment_with_hashtag =
+      comment->second + "%20%23" + hashtag->second;
+
   // If a tweet ID was specified, then quote the original tweet along
   // with the supplied comment; otherwise, just tweet the comment.
   std::string share_url;
   if (!tweet_id->second.empty()) {
     std::string quoted_tweet_url =
         base::StringPrintf("https://twitter.com/%s/status/%s",
                            name->second.c_str(), tweet_id->second.c_str());
-    share_url =
-        base::StringPrintf("https://twitter.com/intent/tweet?url=%s&text=%s",
-                           quoted_tweet_url.c_str(), comment->second.c_str());
+    share_url = base::StringPrintf(
+        "https://twitter.com/intent/tweet?text=%s&url=%s",
+        comment_with_hashtag.c_str(), quoted_tweet_url.c_str());
```

Comment:
nit: why did you switch url and text position?

Reply:
I just wanted the order of the replacement parameters in `StringPrintf` to match in the "if" and "else", so it was more obvious what was going on.

Reply:
make sense 👍